extensions/escape-game/standalone.mjs

```diff
@@ -120,31 +120,41 @@ class Standalone {
       );
     });
     if (config.has('escape-game.doorPosition')) {
-      this.doorCells = [config.get('escape-game.doorPosition')];
+      this.doorCells = config.get('escape-game.doorPosition');
```

Comment:
Consider auto populating `this.doorCells[i]['idx']`.

I would recommend updating the config in hitcon-online too. Update the map if you've time, or leave a TODO.